### DISCUSSION TITLE
galaxy-brain badge

### DISCUSSION THREAD

**Discussion by abir0** [Schweinepriester/github-profile-achievements#521] [Q&A]

**Comment by canvex** [comment]
You should answer other people's question.

**Reply by canvex** [reply]
thanks

**Reply by tom-mcIvor** [reply]
lol that wont work

**Reply by tom-mcIvor** [reply]
you need to get someneelse to mark as answered

**Reply by Trawmoney** [reply]
yes

**Reply by ppnwt** [reply]
Thank You !

**Comment by tuhinmallick** [comment]
Hey! can you please mark my answer

**Reply by tom-mcIvor** [reply]
i think u need to reply to someons post. Can you please mark this as answered :)

**Reply by Trawmoney** [reply]
ok

**Reply by ppnwt** [reply]
Thank You !

**Comment by SusheelThapa** [comment]
When you answer the question asked in any discussion and the answer you answer is marked as answer by the repository maintainer then you can get Galaxy brain badge.
 @abir0

**Reply by pranjal779** [reply]
even the also have to?

**Reply by SusheelThapa** [reply]
@pranjal779  It means alot to me

**Comment by Roberto-Larivera** [comment]
thanks 🧠

**Comment by muzammilali28** [comment]
I still cannot figure out, how does this works ...

**Comment by LauraBigoni** [comment]
For galaxy brain you need to suggest answers to questions posted in discussion. If your answer is helpful then the person who asked the question can mark you answer as the solution. Once you've had 2 questions marked as solutions then you'll get galaxy brain.

Here's the link to the general GitHub community discussions. It might be a good place to start. https://github.com/community

If you found this useful then please mark my answer as a solution so I can get the badge.

**Reply by Trawmoney** [reply]
I found this helpful

**Comment by Trawmoney** [comment]
For galaxy brain you need to suggest answers to questions posted in discussion. If your answer is helpful then the person who asked the question can mark you answer as the solution. Once you've had 2 questions marked as solutions then you'll get galaxy brain.

Here's the link to the general GitHub community discussions. It might be a good place to start. https://github.com/community

If you found this useful then please mark my answer as a solution so I can get the badge
